File: DesignerPro/Map/VB.NET/Map Gallery/ExportViewerFactory.vb
Imports GrapeCity.ActiveReports.Design.Advanced
Imports GrapeCity.ActiveReports.Viewer.Win
Imports GrapeCity.ActiveReports.Viewer.Win.Internal.Export

Friend Class ExportViewerFactory
    Implements IExportViewerFactory
    Public Function CreateExportViewer(viewer As Viewer) As IExportViewer Implements IExportViewerFactory.CreateExportViewer
        Return New ExportViewer(viewer)
    End Function
End Class

File: Advanced/PageAndRDLX/Calendar/TestDesignerPro/ExportViewerFactory.cs
using ActiveReports.Win.Export;
using GrapeCity.ActiveReports.Design.Advanced;
using GrapeCity.ActiveReports.Viewer.Win;
using GrapeCity.ActiveReports.Viewer.Win.Internal.Export;

namespace ActiveReports.Calendar
{
    internal class ExportViewerFactory : IExportViewerFactory
	{
		public IExportViewer CreateExportViewer(Viewer viewer)
		{
			return new ExportViewer(viewer);
		}

    }
}
